.github/workflows/run-e2e-tests.yaml

```diff
@@ -15,46 +15,47 @@ on:
         default: 'main'
         type: string
       wdl-method-version:
-        description: 'Name of branch or release tag in the terra-scientific-pipelines-service repo to use for the test WDL'
-        required: true
+        description: 'Name of branch or release tag in the terra-scientific-pipelines-service repo to use for the test WDL. defaults to latest git release tag if not specified.'
+        required: false
         type: string
 env:
   BEE_NAME: 'teaspoons-${{ github.run_id }}-${{ github.run_attempt}}-dev'
   TOKEN: '${{ secrets.BROADBOT_TOKEN }}'
   RUN_NAME_SUFFIX: ${{ github.event.repository.name }}-${{ github.run_id }}-${{github.run_attempt }}
 jobs:
-  init-github-context:
+  init-github-context-and-params-gen:
     runs-on: ubuntu-latest
     outputs:
-      delete-bee: '${{ steps.extract-inputs.outputs.delete-bee }}'
-      e2e-test-version-ref: '${{ steps.extract-inputs.outputs.e2e-test-version-ref }}'
+      delete-bee: '${{ steps.extract-inputs-and-generate-params.outputs.delete-bee }}'
+      custom-app-version-formatted: '${{ steps.extract-inputs-and-generate-params.outputs.custom-app-version-formatted }}'
+      project-name: '${{ steps.extract-inputs-and-generate-params.outputs.project_name }}'
+      bee-name: '${{ env.BEE_NAME }}'
+      wdl-method-version: '${{ steps.extract-inputs-and-generate-params.outputs.wdl-method-version }}'
     steps:
-      - name: Get inputs or use defaults
-        id: extract-inputs
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get inputs or use defaults, generate other parameters
+        id: extract-inputs-and-generate-params
         run: |
-          echo "delete-bee=${{ inputs.delete-bee || true }}" >> "$GITHUB_OUTPUT"
+          echo "delete-bee=${{ inputs.delete-bee || true }}" >> $GITHUB_OUTPUT
           if [ -z ${{ inputs.custom-app-version }} ]; then
-            echo "e2e-test-version-ref={}" >> "$GITHUB_OUTPUT"
```

Comment:
this was just wrong before, we'd accidentally wired custom-app-version through e2e-test-version-ref.

Reply:
nice catch